packages/coin-support-evm/src/operations/broadcastTransaction/index.ts

```diff
@@ -41,9 +41,9 @@ export const broadcastTransaction = async (
     timestamp: Date.now(),
     blockHeight: -1,
     inputs: [{ address: account.xpubOrAddress, amount: '0', isMine: true }],
-    outputs: [{ ...params.transaction.computedData.output, isMine }],
+    outputs: [{ ...params.transaction.userInputs.outputs[0], isMine }],
     confirmations: 0,
-    accountId: account.__id,
+    accountId: account.parentAccountId ?? account.__id,
     walletId: account.walletId,
     assetId: account.assetId,
     parentAssetId: account.parentAssetId,
```

Comment:
`accountId` and `assetId, parentAssetId` does not seem compatible

Reply:
We don't want to insert token transactions under the token account, transactions are inserted on the root account with the asset id set as a token asset

Reply:
Added `parentAccountId` separately